examplenuget/windows/examplenuget/packages.config

```diff
@@ -1,7 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you remove this file altogether and see if everything still works? I don't think it's necessary anymore.